spec/features/admin/add_custom_organisation_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'support/confirmation_use_case'
+
+describe 'adding a custom organisation name' do
+  include_examples 'confirmation use case spy'
+
+  let!(:admin_user) { create(:user, super_admin: true) }
+
+  context 'when visiting the custom organisations page' do
+    before do
+      sign_in_user admin_user
+      visit admin_custom_organisations_path
+    end
+
+    it 'will show the add custom organisations page' do
+      expect(page.body).to have_content('Allow an organisation access to the admin platform')
+    end
+
+    it 'will redirect you to organisations page once a custom org is added' do
+      fill_in "Enter the organisation's full name", with: 'Custom Org name'
+      expect do
+        click_on 'Allow organisation'
+        .to change { CustomOrganisationName.count }.by(1)
+      end
+    end
+
+    context 'sign out and find custom organisations' do
+      before do
+        sign_out
```

Comment:
The reason you have to sign out here is because this context is nested in the one above. I don't think you need it nested. You are currently running lines 9 - 12 before this context which doesn't need it.